common/src/main/java/org/jboss/hal/testsuite/creaper/AddJMSBridge.java

```diff
@@ -0,0 +1,236 @@
+import org.wildfly.extras.creaper.core.online.operations.Values;
+
+/**
+ * Creaper command for adding JMS Bridge. Only mandatory attributes are covered.
+ * @see <a href="https://docs.jboss.org/author/display/WFLY8/Messaging+configuration#Messagingconfiguration-JMSBridge">https://docs.jboss.org/author/display/WFLY8/Messaging+configuration#Messagingconfiguration-JMSBridge</a>
+ *
+ * @author Jan Kasik <jkasik@redhat.com>
+ *         Created on 9/19/16.
+ */
+public final class AddJMSBridge implements OnlineCommand {
+
+    private String name;
+    private boolean replaceExisting;
+    private QualityOfService qualityOfService;
+    private int failureRetryInterval;
+    private int maxRetries;
+    private int maxBatchSize;
+    private int maxBatchTime;
+    private String sourceConnectionFactory;
+    private String sourceDestination;
+    private Values sourceContext;
+    private String targetConnectionFactory;
+    private String targetDestination;
+    private Values targetContext;
+
+    private AddJMSBridge(Builder builder) {
+        this.name = builder.name;
+        this.replaceExisting = builder.replaceExisting;
+        this.qualityOfService = builder.qualityOfService;
+        this.failureRetryInterval = builder.failureRetryInterval;
+        this.maxRetries = builder.maxRetries;
+        this.maxBatchSize = builder.maxBatchSize;
+        this.maxBatchTime = builder.maxBatchTime;
+        this.sourceConnectionFactory = builder.sourceConnectionFactory;
+        this.sourceDestination = builder.sourceDestination;
+        this.sourceContext = builder.sourceContext;
+        this.targetConnectionFactory = builder.targetConnectionFactory;
+        this.targetDestination = builder.targetDestination;
+        this.targetContext = builder.targetContext;
+    }
+
+    @Override
+    public void apply(OnlineCommandContext ctx) throws Exception {
+        Address address = Address.subsystem("messaging-activemq").and("jms-bridge", name);
+        Operations operations = new Operations(ctx.client);
+
+        if (replaceExisting) {
+            operations.removeIfExists(address);
+        }
+
+        operations.add(address, Values.of("quality-of-service", qualityOfService.getStringValue())
+                .and("failure-retry-interval", failureRetryInterval)
+                .and("max-retries", maxRetries)
+                .and("max-batch-size", maxBatchSize)
+                .and("max-batch-time", maxBatchTime)
+                .and("source-connection-factory", sourceConnectionFactory)
+                .and("source-destination", sourceDestination)
+                .andObject("source-context", sourceContext)
+                .and("target-connection-factory", targetConnectionFactory)
+                .and("target-destination", targetDestination)
+                .andObject("target-context", targetContext));
+    }
+
+    @Override
+    public String toString() {
+        return null;
```

Comment:
Its OK to remove `toString()` completely? I have no idea how this got into the code :)